src/utils.js

```diff
@@ -61,6 +61,7 @@ export const postParams = ({
          method: "POST",
          headers: {
            "Content-Type": "application/json",
+           "Accept-Language": "en",
```

Comment:
@saumier Please let me now if you want to change the default language to `fr`.

Reply:
"en" is good for now.